### ISSUE TITLE
Add: Redmatter blast

### ISSUE THREAD

**Issue by DarkGuardsman** [BuiltBrokenModding/ICBM#173]
Black hole missile from 1.6.4 version

**Comment by jeg623** [comment]
yes please, but don't nerf it again. Actually it was not nerfed but made slower.

**Comment by DarkGuardsman** [comment]
Most likely it was made slower to solve CPU issues. As well it will not be nerfed but also will not be the same as the old version. The plan is to have it scaled to size of the explosive and amount of redmatter used in the recipe.
